.travis.yml

```diff
@@ -8,6 +8,7 @@ services:
 install:
     - sudo pip install -q sphinx --use-mirrors
     - wget -q -O conf.py https://raw.githubusercontent.com/ongr-io/docs-aggregator/master/source/conf-travis.py
+    - composer self-update
```

Comment:
Current composer version in travis works a lot slower than newest one and builds often fail with:
```
No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.
```